jackmatchmaker/__init__.py

```diff
@@ -173,6 +175,10 @@ def reg_callback(self, port_id, action, *args):
         log.debug("New port registered: %s", jacklib.port_name(port))
         self._refresh()
 
+    def error_callback(self, error):
+        error = error.decode('utf-8')
```

Comment:
Please add `errors=ignore`. You never know what strange string Jack might give us.

Reply:
I see, done.